.github/workflows/autobuild.yml

```diff
@@ -62,15 +62,21 @@ jobs:
       publish_to_release:           ${{ steps.get-build-vars.outputs.PUBLISH_TO_RELEASE }}
       upload_url:                   ${{ steps.create-release.outputs.upload_url }}
       build_version:                ${{ steps.get-build-vars.outputs.BUILD_VERSION }}
+    env:
+      release_changelog_path:       ./.github_release_changelog.md
 
     steps:
       - name:                       Checkout code
         uses:                       actions/checkout@v2
 
-      - name:                       Determine release version, type and prerelease variables and generate Changelog
-        run:                        python3 ${{ github.workspace }}/.github/actions_scripts/analyse_git_reference.py
+      - name:                       Determine release version, type and prerelease variables
+        run:                        ./.github/autobuild/get_build_vars.py
         id:                         get-build-vars
 
+      - name:                       Extract Changelog for the Github release body
+        if:                         steps.get-build-vars.outputs.PUBLISH_TO_RELEASE == 'true'
+        run:                        ./.github/autobuild/extractVersionChangelog.pl ChangeLog $(grep -oP 'VERSION = \K\w[^\s\\]*' Jamulus.pro) > ${{ env.release_changelog_path }}
```

Comment:
Not that happy with a regex and grep of the version here. Don't we have the version somewhere else?

Reply:
The very version we need here is available in the script, but currently isn't output. We can change that, but it would be a more distributed change. See here:
https://github.com/jamulussoftware/jamulus/pull/2655/files#diff-54b7cee412550302a3e076162e611b9719407b264c79553452499450c54a07e5R52 https://github.com/jamulussoftware/jamulus/pull/2655/files#diff-55d631a22c9e350832273ee0abf8a6013435fa6051789cbe311b981b2ec0f348R88

Would you prefer that?

Reply:
That's rather why I was wondering if returning the release type from `get_version_from_jamulus_pro()` might help elsewhere.

Reply:
Yes, personally, I'd prefer this since it feels cleaner.

Reply:
Ok, updated and re-tested on my fork (updated the link in the PR description).